build_node/mock/mock_config.py

```diff
@@ -158,7 +160,8 @@ def __init__(self, target_arch, legal_host_arches=None,
         self.__config_opts = {'target_arch': target_arch,
                               'legal_host_arches': legal_host_arches,
                               'chroot_setup_cmd': chroot_setup_cmd,
-                              'dist': dist, 'releasever': releasever}
+                              'dist': dist, 'releasever': releasever,
+                              'basedir': basedir}
```

Comment:
Looks like that we use here `None` by default instead of `/var/lib/mock/` or `mock` use default path even if `basedir = None`?

Reply:
Yes, I ensured that it uses Mock's default value for basedir, and https://github.com/AlmaLinux/albs-node/blob/bdfd8756d8bcab3cd31a34b0000138698a84f939/build_node/mock/mock_config.py#L373 takes care of it

Reply:
oh, okay, missed this one, thanks :+1:

Reply:
Yeah, when `basedir` is `None`, it'll use mock's default, whatever it is. According to mock's docs, it's `/var/lib/mock/`. That's why I indicated such folder.